rules/xcodeproj.bzl

```diff
@@ -269,6 +269,66 @@ def _joined_header_search_paths(hmap_paths):
     header_search_paths.append("\"$BAZEL_WORKSPACE_ROOT\"")
     return " ".join(header_search_paths)
 
+def _gather_asset_sources(target_info, path_prefix):
+    """Helper method gather asset sources (non-code resources) based on its type or special names
+
+    Args:
+        target_info: containing asset_srcs and build_files that we compose the list from
+        path_prefix: any prefix needed to correctly resolve a relative path in xcode proj
+    Returns:
+        array of dictionry, each dict represents a Target Resource described in XcodeGen:
```

Comment:
Thanks for adding these docs!

spelling nit: "array of dictionaries, each dict..."